### DISCUSSION TITLE
List of terms used in corenlp when analysing sentences

### DISCUSSION THREAD

**Discussion by dinadot** [stanfordnlp/CoreNLP#1337] [Q&A]
I am after a list of all the terms used in this program. I have worked out many myself but thought there may be a list somewhere. ie- VB- verb, DET- determiner
There are some terms I am unaware of ie case.

Thank you

**Comment by AngledLuffa** [comment]
VB, DET, etc are from the PTB POS tags

https://www.ling.upenn.edu/courses/Fall_2003/ling001/penn_treebank_pos.html
case and other dependency terms are from UD

https://universaldependencies.org/guidelines.html
